Moco/exampleSquat/exampleSquat.m

```diff
@@ -0,0 +1,339 @@
+modelProcessor = ModelProcessor(inModel);
+jointsToWeld = StdVectorString();
+jointsToWeld.add('mtp_r');
+jointsToWeld.add('mtp_l');
+modelProcessor.append(ModOpReplaceJointsWithWelds(jointsToWeld));
+modelProcessor.append(ModOpReplaceMusclesWithDeGrooteFregly2016());
+modelProcessor.append(ModOpIgnorePassiveFiberForcesDGF());
+modelProcessor.append(ModOpTendonComplianceDynamicsModeDGF('implicit')); 
+
+% Set the settings for the MocoTrack problem.
+track.setModel(modelProcessor);
+track.setStatesReference(tableProcessor);
+track.set_states_global_tracking_weight(stateTrackingWeight);
+track.set_allow_unused_references(true);
+track.set_track_reference_position_derivatives(true);
+track.set_apply_tracked_states_to_guess(true);
+track.set_initial_time(initialTime);
+track.set_final_time(finalTime);
+track.set_mesh_interval(meshInterval);
+study = track.initialize();
+problem = study.updProblem();
+
+
+% Goals
+% =====
+
+% Tracking
+% --------
+% Set different tracking weights for states (weights for states not 
+% explicitly set here have a default value of 1.0)
+stateTrackingGoal = MocoStateTrackingGoal.safeDownCast(problem.updGoal('state_tracking'));
+stateTrackingGoal.setWeightForState('/jointset/ground_pelvis/pelvis_tilt/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/ground_pelvis/pelvis_tilt/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/back/lumbar_extension/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/back/lumbar_extension/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/hip_r/hip_flexion_r/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/hip_r/hip_flexion_r/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/hip_l/hip_flexion_l/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/hip_l/hip_flexion_l/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/walker_knee_r/knee_angle_r/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/walker_knee_r/knee_angle_r/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/walker_knee_l/knee_angle_l/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/walker_knee_l/knee_angle_l/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/ankle_r/ankle_angle_r/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/ankle_r/ankle_angle_r/value', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/ankle_l/ankle_angle_l/speed', 10.0);
+stateTrackingGoal.setWeightForState('/jointset/ankle_l/ankle_angle_l/value', 10.0);
+
+% Periodicity
+% -----------
+% This goal enforces periodicity (i.e., enforce that certain values are
+% the same at the beginning of the simulation as the end of the
+% simulation).
+periodicityGoal = MocoPeriodicityGoal('periodicityGoal');
+problem.addGoal(periodicityGoal);
+model = modelProcessor.process();
+model.initSystem();
+
+% Periodic coordinate values (except for pelvis_tx) and speeds.
```

Comment:
not obvious why you'd exclude tx and not tz?

Reply:
updated with explanation. although this works as is, could be good later to check the effect of also adding pelvis_tx periodicity and simplify the example if it doesn't have to be left out

Reply:
helpful explanation. thanks. I get it now, and the choice makes sense.